assets/css/editor.scss

```diff
@@ -1,5 +1,6 @@
 // Import the woocommerce components stylesheet
 @import "~@woocommerce/components/build-style/style.css";
+@import "style.scss";
```

Comment:
Can you add a quick comment saying why this is necessary? I see why from the PR description but for future reference (since this is kind of an anti-pattern).

Reply:
Actually, this doesn't totally work – `./style.scss` isn't the only "front end" style file, each block could also have a local style.scss, and it's all built into `build/style.css` – for example, the featured product block does, but now those CSS rules are not included in the editor, which causes the featured product block to do this:

![576-featured-product](https://user-images.githubusercontent.com/541093/58050064-53fe8280-7b1c-11e9-96da-a22a1ff5cc72.png)

But the frontend is still fine because we're correctly enqueuing the build/style.css file there.